docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { Image, Button } from '@stardust-ui/react'
+
+import * as React from 'react'
+import * as _ from 'lodash'
+
+export interface GridPickerItemProps {
+  as?: keyof React.ReactHTML
+  title?: string
+  imageSrc: string
+  onClick?: (e: React.SyntheticEvent, props: GridPickerItemProps) => void
+}
+
+const imageStyles = {
+  width: '100%',
+}
+
+const imageButtonStyles = {
+  minWidth: '56px',
+  height: '56px',
+  padding: '0',
+  background: '#fff',
```

Comment:
How do you handle themeing (dark / contrast) for values like this? Or is that not something we worry about in prototypes?

Reply:
yes, exactly, this prototype just shows how things can be combined together from stardust component to build an accessible experience, so I think we can speak on theming here.